composer.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "fusesource/stomp-php",
     "description": "stomp support for PHP",
+    "abandoned": true,
```

Comment:
Is the `abandoned` flag contained in the composer schema? 
I thought it's only controlled by the packagist project homepage.